Iteam1337/predictivemovement
# feat: 🎸 add postgres backup deployment

- it adds a `postgres-backup` container in the stack that will backup Postgres every 6 hours

### How to test
- it's already deployed to the dev environment (`predictivemovement-dev` namespace)
- exec into the `postgres` pod and run a `DROP` command on a table or db via psql (I had to exec to be able to run drop command, somehow my sql client didn't allow me to do it if I connect it to postgres via port-forwarding)
- exec into the `postgres-backup` pod and run `/restore.sh /backup/latest.psql.gz` (or choose a different backup you want) 
- verify again in `postgres` that the table/db or whatever you've destroyed has been restored
